docs/resources/file.md.erb

```diff
@@ -87,6 +87,22 @@ The `mode` property tests if the mode assigned to the file matches the specified
 
     its('mode') { should cmp '0644' }
 
+You must write the numeric mode specification starting with zero, to tell InSpec
+the value is [octal](https://en.wikipedia.org/wiki/Leading_zero#0_as_a_prefix).
```

Comment:
InSpec [octal](https://en.wikipedia.org/wiki/Leading_zero#0_as_a_prefix) values begin the numeric mode specification with zero.